docs/development/debugging/README.md

```diff
@@ -0,0 +1,5 @@
+# Debugging
+
+## Contents
+* [Visual Studio][visualstudio.md]
+* [x64dbg][x64dbg.md]
```

Comment:
Typo

```suggestion
* [Visual Studio](visualstudio.md)
* [x64dbg](x64dbg.md)
```

Reply:
fixed in 0bf60015df4b9b245fd0ee2827ce99af134b5ebc